html/cross-origin-embedder-policy/none.https.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<meta name="timeout" content="long">
+<title>Cross-Origin-Embedder-Policy header and nested navigable resource without such header</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=/common/utils.js></script> <!-- Use token() to allow running tests in parallel -->
+<div id=log></div>
+<script>
+async_test(t => {
+  const frame = document.createElement("iframe");
+  t.add_cleanup(() => frame.remove());
+  frame.onload = t.step_func_done(() => {
+    assert_not_equals(frame.contentDocument, null);
+  });
+  frame.src = "/common/blank.html";
+  document.body.append(frame);
+  assert_equals(frame.contentDocument.body.localName, "body");
+}, `"none" top-level: navigating a frame to "none" should succeed`);
+
+async_test(t => {
+  const frame = document.createElement("iframe");
+  t.add_cleanup(() => frame.remove());
+  const blank = "/common/blank.html";
+  let firstNavOk = false;
+  frame.onload = t.step_func(() => {
+    assert_not_equals(frame.contentDocument, null);
+    firstNavOk = true;
+  });
+  t.step_timeout(() => {
+    assert_true(firstNavOk);
+    assert_not_equals(frame.contentDocument, null);
+    assert_equals(frame.contentWindow.location.pathname, blank);
+    t.done();
+  }, 500);
```

Comment:
It is a risky setup. `frame.contentWindow` might not be updated well in 500 ms for some environment (e.g., treeherder). My debug nightly will fail to update in time